database/migrations/2018_04_16_162649_basic_tables.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class BasicTables extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('workflows', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger("authorId");
+            $table->timestamp("created")->useCurrent();
+            $table->timestamp("updated")->nullable();
+            $table->string("languageCode",2);
+            $table->string("title",30);
+            $table->string("description",5000)->nullable();
+            $table->boolean("isDraft")->nullable()->default(true);
```

Comment:
Why would you make this `nullable` and give it a default value of `true`? That doesn't make a lot of sense.

Reply:
True. Fixed in a new migration in the database branch.